### PR TITLE
[mime-db] Add exported object

### PR DESCRIPTION
**Pull request by LinusU**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/jshttp/mime-db#usage https://github.com/jshttp/mime-db#data-structure
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Comment by LinusU** [comment]
I fixed it directly after but forgot to push 🤦‍♂

**Comment by LinusU** [comment]
Okay, this should be good to go now 💪 🚀 

ping @RyanCavanaugh, @orta, @sandersn: I'm touching a lot of libs here so I think it would be great if someone from the team could step in and look at this 😅

**Comment by LinusU** [comment]
ping @rbuckton @DanielRosenwasser, would anyone be able to check on this one? Thanks ❤️

File: types/mime-db/index.d.ts [hunk 1]
```diff
@@ -1,21 +1,33 @@
 // Type definitions for mime-db 1.27
 // Project: https://github.com/jshttp/mime-db
 // Definitions by: AJP <https://github.com/AJamesPhillips>
+//                 Linus Unnebäck <https://github.com/LinusU>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.2
+// TypeScript Version: 3.4
```
Comment (SimonSchick) [inline types/mime-db/index.d.ts]: This change seems redundant.

**Review by SimonSchick — CHANGES_REQUESTED** [review]
I disagree with the TS version increase, you don't seem to be using any TS 3.4 features?

File: types/mime-db/index.d.ts (hunk 1 above)
Comment (SimonSchick) [reply types/mime-db/index.d.ts]: To clarify, you should only bump it as high as you _need_ to make that required changes, if you have to mass bump other packages that's usually a bad sign.

File: types/mime-db/index.d.ts [hunk 2]
```diff
@@ -1,21 +1,33 @@
...
-    charset?: string;
+declare namespace database {
+    /**
+     * @see {@link https://github.com/jshttp/mime-db#data-structure}
+     */
+    interface MimeEntry {
+        /** Where the mime type is defined. If not set, it's probably a custom media type. */
+        readonly source?: string;
+        /** Known extensions associated with this mime type. */
+        readonly extensions?: readonly string[];
```
Comment (SimonSchick) [inline types/mime-db/index.d.ts]: You probably want to use `ReadonlyArray<string>` instead as that's more compatible.

Comment (LinusU) [reply types/mime-db/index.d.ts]: I wasn't aware that `ReadonlyArray` was available before `readonly ...[]`, thanks!

**Comment by LinusU** [comment]
Replaced by https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43066, which uses `ReadonlyArray` in order to maintain TypeScript 2.2 compatibility...

**Pull request closed without merging**